clang/include/clang/Basic/AArch64SVEACLETypes.def

```diff
@@ -107,6 +107,15 @@
   AARCH64_VECTOR_TYPE(Name, MangledName, Id, SingletonId)
 #endif
 
+#ifndef AARCH64_SCALAR_TYPE
+#define AARCH64_SCALAR_TYPE(Name, MangledName, Id, SingletonId) \
+  SVE_TYPE(Name, Id, SingletonId)
+#endif
+
+#ifndef AARCH64_SCALAR_TYPE_MFLOAT
+#define AARCH64_SCALAR_TYPE_MFLOAT(Name, MangledName, Id, SingletonId, NumEls, ElBits, NF) \
+  AARCH64_SCALAR_TYPE(Name, MangledName, Id, SingletonId)
+#endif
```

Comment:
Hopefully this isn't a bum steer but now everything has settled down I think these are no longer necessary and you can use `AARCH64_VECTOR_TYPE_MFLOAT` to define the scalar type as well because its internal representation is still a vector.

Reply:
Yes, I agree with you that we dont need a scalar type anymore, because the MFloat8_t will also be a vector.